integration/form-test.ts

```diff
@@ -576,3 +576,53 @@ test.describe("Forms", () => {
     });
   });
 });
+
+test.describe("Forms in an app with custom defineRoutes", () => {
+  let fixture: Fixture;
+  let appFixture: AppFixture;
+
+  test.beforeAll(async () => {
+    fixture = await createFixture({
+      template: "node-custom-routes-template",
```

Comment:
@penspinner Is there a specific reason you're not just using `node_template` (the default) here?

You could also use [Pathless Layout Routes](https://remix.run/docs/en/v1/guides/routing#pathless-layout-routes) if you want to remove certain sections from the URL.

Reply:
I have to test for the case where the route does not end in `"/index"` but `useFormAction` still adds the `"?index"` to the form action. Unless I'm mistaken, I think this only happens when the user is defining custom routes. There doesn't seem to be a way to pass a `defineRoutes` to `createFixture`, so I had to create a new template to test this.

Reply:
You can always create a `remix.config.js` file entry in your test if you want change the behaviour of it.

Reply:
Oh, I see what you mean now. Didn't know we could define the Remix config in the `createFixture`.